app/uk/gov/hmrc/helptosavefrontend/connectors/HelpToSaveConnector.scala

```diff
@@ -124,21 +124,24 @@ class HelpToSaveConnectorImpl @Inject() (http: WSHttp)(implicit ec: ExecutionCon
   private val eligibilityURL = s"$helpToSaveUrl/help-to-save/eligibility-check"
 
   private def toEligibilityCheckResponse(eligibilityCheckResponse: EligibilityCheckResponse): Either[String, EligibilityCheckResult] = {
```

Comment:
minor but can you rename `toEligibilityCheckResponse` to `toEligibilityCheckResult`  ?  as the calling line is not clear on this : 
 ```_.parseJson[EligibilityCheckResponse].flatMap(toEligibilityCheckResponse),```